### DISCUSSION TITLE
컴퓨터 밑바닥의 비밀

### DISCUSSION THREAD

**Discussion by sunhpark42** [elegant-functional-2023/secrets-of-computer#2] [Announcements]
- 스터디 시간:  토요일 11시, 테크살롱
- 스터디 시작전 까지
    - 감상평 가볍게 올리기
        - 템플릿: 질문 2개 뽑기 / 없으면 인상깊은 문장 공유하기/질문 미리 작성해오기 등등.
        - 디스커션에 남겨두는게 다같이 보기 좋을 듯!
- 감상평을 안올렸다면?
    - 다음 스터디 시작 전까지, 안읽은 부분 요약해서 깃허브에 올리기.
- 진행 차수는 9주 (스터디 8주, 회고 1주)

스레드로 스터디를 시작하기 전에 바라는 것, 얻어가고 싶은 것을 한마디씩 적어주세요 🤩

**Comment by yunseorim1116** [comment]
### 바라는 것
이번엔 All 출석 ! 항상 그랬듯이 No 지각! 목표 입니다.
완독을 하고 싶습니다. 
모르는게 있으면 부끄러워하지 말고 물어보기!

### 얻어가고싶은 것
저는 현재 CS 베이스가 거의 0에 수렴한다고 자신있게 말할 수 있는데요...^^......
일단 첫 번째는 당연하게도 기본 CS지식을 얻어가고 싶습니다.  
두 번째로는  함께 스터디를 하며 자극을 받고, 멋진 여러분들을 보며 더 열심히 공부해야겠다고 느끼고 싶습니다.

**Comment by ella-yschoi** [comment]
### 바라는 것

- 바보 같은 질문이라도 궁금하면 알아내서 내 것으로 만들기
- 스터디원 분들과 (신뢰를 기반으로,,) 친해지기 🥹

### 얻어가고 싶은 것

- 면접에서 '이 스터디를 통해 무엇을 배웠나요?' 질문을 받으면 2가지 이상 답하기
- 매 챕터 읽고 블로그 포스팅해서 완독하면 6개 이상의 포스팅을 남기기
